optaplanner-core/src/main/java/org/optaplanner/core/impl/domain/locator/Locator.java

```diff
@@ -21,22 +21,20 @@
 import java.util.Map;
 
 import org.optaplanner.core.api.domain.locator.PlanningId;
-import org.optaplanner.core.api.domain.solution.PlanningSolution;
 import org.optaplanner.core.impl.domain.solution.descriptor.SolutionDescriptor;
 import org.optaplanner.core.impl.score.director.ScoreDirector;
 
 /**
- * @param <Solution_> the solution type, the class with the {@link PlanningSolution} annotation
  * @see PlanningId
  * @see ScoreDirector#locateWorkingObject(Object)
  */
-public class Locator<Solution_> {
+public class Locator {
```

Comment:
thanks for pointing this out.

I am ok with it if we change the constructor to Locator(LocationStrategyResolver locationStrategyResolver) (see below)